### DISCUSSION TITLE
Physical Modeling Syth

### DISCUSSION THREAD

**Discussion by Reaper10** [tim-janik/anklang#11] [Ideas]
This is just an idea for anklang is to have an  physical modeling syth after all there isn't an open source  physical modeling syth but there is tools to make one [Faust](https://faust.grame.fr/) and [ChucK ](https://chuck.cs.princeton.edu/) maybe you make a mix of both. 
list of modeling synth

[Friktion ](https://www.youtube.com/watch?v=vUz3WEUJuQs&t=427s)

[Sculpture](https://www.youtube.com/watch?v=z9XNueV_w9I)

[Objekt Modeling Synthesizer](https://www.youtube.com/watch?v=ASE8nDgBEQg&t=176s)

the ui could be like blender  geometry nodes or bitwig grid
![file](https://github.com/tim-janik/anklang/assets/8137561/b77da130-4b6c-4e6b-91cb-6cba5b0f9a2e)
![Bitwig-BWS3-2_Enter-the-Grid](https://github.com/tim-janik/anklang/assets/8137561/f4832bde-83b6-4779-bf36-b9c6cd498a36)
it would be  a moduler syth in module daw
 this would make a great sound design tool

**Comment by tim-janik** [comment]
Please don't abuse the discussion forum as link roaster.

If you want your comments to be taken seriously, actually engage in the discussions in the other threads you started participating in. I.e. answer/reply to comments addressed at you and make an actual effort to *run* Anklang.

Your postings are of no use to the project unless we can have a proper conversation (where questions deserve an answer) and unless you can actually try out the thing we are talking about.

**Reply by Reaper10** [reply]
You are completely right.I Did not illustrate why my idea would have been useful or helpful. There's might be too little too late, but I'm gonna try to illustrate. What I was thinking. Having a physical modeling synthesizer in your daw  With a Physical modeling synthesizer. You do not need to rely and mountains and mountains of samples like you would with a sampler It's not as memory intensive on the CPU of the computer sample. And another benefit of a physical model synthesizer is for those instruments.The user wants to use or build that they don't have access too good quality samples.To you wrote a sampler. And a physical modeling synthesizer would be great for sound design. Like, for example, you could make a pretty convincing.Synthesizer synthesizer to recreate a prepared piano My guide pointed out not everyone has the access or no. How to prepare a piano to record samples of it for the sampler? Why you could use the physically model synthesizer to recreate that no one builds anymore. One other good use for a physical model synthesizer is for those moments. Where you do have the instrument. You're modeling, but you don't have it with you. So you can use the physical model synthesizer as a temp.Until you can get the actual instrument recorded
